lib/seahorn/BvOpSem2Context.hh

```diff
@@ -729,6 +731,14 @@ public:
   /// \brief given a properly encoded pointer Expr \p p , return the raw
   /// expression representing memory address only
   virtual Expr ptrToAddr(Expr p) = 0;
+
+  /// \brief given an Expression \p e , return true if \p e has expected
+  /// encoding of a MemValTy
+  virtual bool isMemVal(Expr e) = 0;
```

Comment:
This is not needed in the interface (context) class since no clients of context use this. Please remove this method.